### PR TITLE
Add yaml & ruby modes

### PR DESCRIPTION
**Pull request by Robdel12**

### CODE REVIEW

File: frontmacs-ruby.el [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+(add-to-list 'auto-mode-alist '("Thorfile\\'" . ruby-mode))
+(add-to-list 'auto-mode-alist '("Vagrantfile\\'" . ruby-mode))
+(add-to-list 'auto-mode-alist '("\\.jbuilder\\'" . ruby-mode))
+(add-to-list 'auto-mode-alist '("Podfile\\'" . ruby-mode))
+(add-to-list 'auto-mode-alist '("\\.podspec\\'" . ruby-mode))
+(add-to-list 'auto-mode-alist '("Puppetfile\\'" . ruby-mode))
+(add-to-list 'auto-mode-alist '("Berksfile\\'" . ruby-mode))
+(add-to-list 'auto-mode-alist '("Appraisals\\'" . ruby-mode))
+
+(provide frontmacs-ruby)
```
Comment (cowboyd) [inline frontmacs-ruby.el]: this needs to be a symbol, no?

```emacs-lisp
(provide 'frontmacs-ruby)
```

File: frontmacs-yaml.el [hunk 2]
```diff
@@ -0,0 +1,5 @@
+(require 'yaml-mode)
+
+(add-to-list 'auto-mode-alist '("\\.yml\\'" . yaml-mode))
+
+(provide frontside-yaml)
```
Comment (cowboyd) [inline frontmacs-yaml.el]: Should be:

```
(provide 'frontmacs-yaml)
```

I think?

**Review by cowboyd — CHANGES_REQUESTED** [review]

File: frontmacs-pkg.el [hunk 3]
```diff
@@ -23,6 +23,7 @@
     (emmet-mode "1.0.8")
     (smartparens "1.9.0")
     (web-mode "14.1")
+    (yaml-mode "0.0.13")
```
Comment (cowboyd) [inline frontmacs-pkg.el:26]: I think we want to include `ruby-mode` from melpa. Does that work?

Comment (Robdel12) [reply frontmacs-pkg.el:26]: ruby-mode doesn't exist on melpa

Comment (Robdel12) [reply frontmacs-pkg.el:26]: image
image

**Review by cowboyd — APPROVED** [review]

**Pull request merged**